easysocial/easysocial/share.php

```diff
@@ -50,6 +50,18 @@ public function get()
 	 * @since 1.0
 	 */
 	public function post()
+	{
+		$this->plugin->setResponse($this->postStory());
+	}
+
+	/**
+	 * Method description
+	 *
+	 * @return  mixed
+	 *
+	 * @since 1.0
+	 */
+	public function postStory()
```

Comment:
@Pratikshakhandagale  - except `GET`,  `POST` and other HTTP request methods, all other methods should be private.